ambry-replication/src/main/java/com.github.ambry.replication/BlobIdTransformer.java

```diff
@@ -156,6 +160,12 @@ private Message newMessage(InputStream inputStream, StoreKey newKey, MessageInfo
     storeKeyFactory.getStoreKey(new DataInputStream(inputStream));
     BlobId newBlobId = (BlobId) newKey;
 
+    if (headerFormat.hasLifeVersion() && headerFormat.getLifeVersion() != oldMessageInfo.getLifeVersion()) {
+      // The original Put buffer might have lifeVersion as 0, but the message info might have a higher lifeVersion.
+      logger.trace(
```

Comment:
Not warn?

Reply:
this is really not important message, I used to use warn just for debugging purpose.